modules/vaos/app/controllers/vaos/v2/slots_controller.rb

```diff
@@ -4,7 +4,10 @@ module VAOS
   module V2
     class SlotsController < VAOS::V0::BaseController
       def index
-        response = systems_service.get_available_slots(slots_params)
+        response = systems_service.get_available_slots(location_id: location_id,
+                                                       clinic_id: clinic_id,
+                                                       start_dt: start_dt,
+                                                       end_dt: end_dt)
```

Comment:
I was wondering why these params got split out to be individually passed in? (if it works it works, I've just had our code checks raise concern when I pass in a large amount of method params before which is why I'm wondering)

Reply:
they were split to uncouple the http origin from the service layer. The controller knows that the data comes from an http session. The service layer should not need to know that the data came from ActionController::Parameters.  

In Ruby named parameters can also be passed in as a Hash, in addition to being passed in individually.

Reply:
Gotcha that all makes sense, thanks for the insight